Dockerfile

```diff
@@ -3,9 +3,9 @@
 ####################################################################################################
 
 ARG PYTHON_VERSION=3.11
-FROM python:${PYTHON_VERSION}-slim-bookworm AS builder
+FROM python:${PYTHON_VERSION}-bookworm AS builder
```

Comment:
The `-slim-bookworm` here replaced with `-bookworm`. However, the same `-slim-bookworm` at line 31 remains unchanged. Why is the difference?